### PR TITLE
Add security focused paragraphs

### PR DESCRIPTION
**Pull request by inghylt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,59 @@ Table of Contents
...
+   mind during Development Stage
+  * Perform security testing during Testing Stage
+  
+  More information about secure SDLC:
+  https://owasp.org/www-project-integration-standards/writeups/owasp_in_sdlc/
+
+###Speaking about security….
+To be safe while navigating the wild west that modern computing can
+be, make sure to:
+* Keep software up-to-date (Turn on Automatic Updates for your
```
Comment (kusalananda) [inline README.md:97]: Lower-case text in parenthesis.

File: README.md [hunk 2]
```diff
@@ -61,6 +61,59 @@ Table of Contents
...
+ operating system/programs)
+* Use a password manager (third-party or browser’s built-in depending
+ on your needs) to facilitate strong unique passwords. Do not reuse
+ passwords.
+* Use two-factor authentication, e.g. Authy or Google Authenticator.
+ This is supported by many services such as GitHub, Slack, Gmail etc.
+ In the not so unlikely event that someone does obtain your password
+ , two-factor authentication can still protect you/ sensitive
```
Comment (kusalananda) [inline README.md:105]: Dangling comma, and spacing in general here.

File: README.md [hunk 3]
```diff
@@ -61,6 +61,59 @@ Table of Contents
 
 ## Things to be aware of when writing code
 
+###Writing secure software
```
Comment (kusalananda) [inline README.md:64]: These added sections have too many slashes, e.g., "Use/Implement tests/build processes/libraries", "protect you/ sensitive information", "what you click/attach" etc.  Spell out "and" and "or" properly to make the text flow better.

File: README.md [hunk 4]
```diff
@@ -61,6 +61,59 @@ Table of Contents
...
+* Use a password manager (third-party or browser’s built-in depending
+ on your needs) to facilitate strong unique passwords. Do not reuse
+ passwords.
+* Use two-factor authentication, e.g. Authy or Google Authenticator.
+ This is supported by many services such as GitHub, Slack, Gmail etc.
+ In the not so unlikely event that someone does obtain your password
+ , two-factor authentication can still protect you/ sensitive
+ information
+* Always lock your computer when unattended (even if you are just going
+ for a quick coffee :)))
```
Comment (kusalananda) [inline README.md:108]: Avoid smiley.

File: README.md [hunk 5]
```diff
@@ -61,6 +61,59 @@ Table of Contents
...
+* Configure so your device is automatically locked after 1-5 minutes
+* Remove unnecessary programs from your computer to reduce attack
+ surface
+* Encrypt your drives, especially external
```
Comment (kusalananda) [inline README.md:112]: Check consistent use of full stop at the end of list items.

File: README.md [hunk 6]
```diff
@@ -61,6 +61,59 @@ Table of Contents
...
+analysis should be employed at the early stage of a project (it is
+usually way more expensive and complicated at a late stage) and
+continuously carried out until the end. 
+
+The security risks will differ vastly between project, but general
+guidelines are:
+
+* Educate yourself, research on what security risks are related to
+ the different components of your project. Some example resources are:
+  * https://bestpractices.coreinfrastructure.org/en
```
Comment (kusalananda) [inline README.md:77]: Consider using link texts rather than bare URLs.

File: README.md [hunk 7]
```diff
@@ -61,6 +61,59 @@ Table of Contents
...
+* Don’t blindly trust out-of-the-box software and default
+ configurations, e.g. popular Docker images, aws services, etc. 
+ Malicious software can be present and default configurations usually
+ have simplicity as the primary goal, not security.
+* Incorporate security in the entire Software Development Lifecycle:
+  * Include security considerations in Requirements Gathering & Analysis
+  * Threat model during Design Stage
+  * Use/Implement tests/build processes/libraries with security in
+   mind during Development Stage
+  * Perform security testing during Testing Stage
```
Comment (kusalananda) [inline README.md:89]: "Testing Stage" and "Design Stage" etc. sounds like you're referring to a formal process. What is that process? If not referring to a formal process, use lower-case.

**Review by kusalananda — DISMISSED** [review]

**Comment by jhagberg** [comment]
Should we add something about ISO27001 ?

**Comment by viklund** [comment]
I really like the content here, much needed.

I do agree with @kusalananda about the comments, especially the desing phase/stage things, since we don't have that formalised it's a bit tricky to reference here, maybe instead just say "when desiging" and "when testing" or something like that.

I think we should wait with ISO27001 here for later.

big 👍

File: README.md [hunk 8]
```diff
@@ -61,6 +61,75 @@ Table of Contents
 
 ## Things to be aware of when writing code
 
+###Writing secure software
```
Comment (airenzp) [inline README.md:64]: You need to add a space here "###Writing secure software" between the ### and the text. Nice section, but I would not put it in the beginning of the ## Things to be aware of when writing code, maybe after Testing?

Comment (inghylt) [reply README.md]: I believe it is important to have it early on in the guidelines. A common problem related to security is that it is not thought of early in the process, but really it should be taken into consideration in the very beginning. Having it early in the guidelines will hopefully make more people think of it. To me it makes sense to have it under "Things to be aware of when writing code", but I'm also up for moving it to a separate section

**Review by norling — APPROVED** [review]
I agree with Andreas comments, apart from that it looks good!

**Review by jbygdell — APPROVED** [review]

**Review by viklund — APPROVED** [review]

**Review by nanjiangshu — APPROVED** [review]

**Review by jhagberg — APPROVED** [review]

**Review by danr — APPROVED** [review]

**Review by kostas-kou — APPROVED** [review]

**Pull request merged**